zigpy/config/__init__.py

```diff
@@ -81,6 +83,7 @@
     vol.Optional(CONF_OTA_DIR, default=CONF_OTA_OTAU_DIR_DEFAULT): vol.Any(None, str),
     vol.Optional(CONF_OTA_IKEA, default=CONF_OTA_IKEA_DEFAULT): cv_boolean,
     vol.Optional(CONF_OTA_IKEA_URL): vol.Url(),
+    vol.Optional(CONF_OTA_INOVELLI, default-CONF_OTA_INOVELLI_DEFAULT): cv_boolean,
```

Comment:
This line has a syntax error:

```Python
    vol.Optional(CONF_OTA_INOVELLI, default-CONF_OTA_INOVELLI_DEFAULT): cv_boolean,
                                           ^
```

Reply:
Thank you for catching that...I'd edited this line in by hand vs copy/pasting, which is why it also didn't error locally as you pointed out.